static/app/stores/modalStore.tsx

```diff
@@ -44,9 +50,23 @@ const storeConfig: ModalStoreDefinition = {
   },
 
   openModal(renderer: Renderer, options: ModalOptions) {
-    this.state = {renderer, options};
+    this.state = {...this.state, renderer, options};
     this.trigger(this.state);
   },
+
+  setPauseFocusTrap(fx: State['pauseFocusTrap'] | null) {
```

Comment:
`null` is part of the State definition

```suggestion
  setPauseFocusTrap(fx: State['pauseFocusTrap']) {
```